rollingpin/args.py

```diff
@@ -243,3 +257,28 @@ def construct_canonical_commandline(config, args):
             arg_list.extend(command)
 
     return " ".join(arg_list)
+
+
+def _get_available_profiles(profile_dir):
+    profiles = []
+    for f in os.listdir(profile_dir):
+        if f.endswith(".ini"):
+            profile_name, _ = f.split(".", 1)
+            profiles.append(profile_name)
```

Comment:
I used this over the `glob` module because `listdir` *only* returns the filename where `glob` returns the full path.  The resulting parsing would have been something like `f = p.split('/')[-1]` in place of the conditional.